androidTEst/androidTEst/Screens/AllFilesScreen.cs

```diff
@@ -0,0 +1,116 @@
+﻿using androidTEst.Extension;
+using Aquality.Appium.Mobile.Elements.Interfaces;
+using Aquality.Appium.Mobile.Screens;
+using OpenQA.Selenium;
+
+namespace Aquality.Appium.Mobile.Template.Screens.Login
+{
+    public class AllFilesScreen : Screen
+    {
+        private readonly IButton AddNewFileButt = ElementFactory.GetButton(By.Id("com.nextcloud.client:id/fab_main"), "AddNewFileButt");
+        private readonly IButton MenuButt = ElementFactory.GetButton(By.Id("com.nextcloud.client:id/menu_button"), "MenuButt");
+        private readonly IButton AllFilesButt = ElementFactory.GetButton(By.Id("com.nextcloud.client:id/sort_list_button_group"), "AllFilesButt");
+
+        // regardless of the language of the system/application, the text "текстовый документ" doesn't change. (see ./Screenshots)
+        private readonly IButton CreateDoc = ElementFactory.GetButton(By.XPath("//android.widget.TextView[@resource-id= 'com.nextcloud.client:id/creator_name' and contains(@text, 'текстовый документ')]"), "CreateDoc");
+        private readonly ITextBox TextFileName = ElementFactory.GetTextBox(By.Id("com.nextcloud.client:id/filename"), "TextFileName");
+        private readonly IButton CreateButton = ElementFactory.GetButton(By.Id("android:id/button1"), "CreateButton");
+        private ILabel MyFile(string fileName) => ElementFactory.GetLabel(By.XPath($"//android.widget.TextView[@resource-id = 'com.nextcloud.client:id/Filename' and contains(@text, '{fileName}')]"), "MyFile");
+        private readonly By ListOfFiles = By.XPath("//android.widget.TextView[@resource-id='com.nextcloud.client:id/Filename']");
+        private IButton MoreMemu(int index) => ElementFactory.GetButton(By.XPath($"(//android.widget.ImageView[@resource-id='com.nextcloud.client:id/overflow_menu'])[{index}]"), "MoreMemu");
```

Comment:
меню лучше описать как отдельную форму и использовать композицию или агрегацию
```suggestion
        private IButton MoreMenu(int index) => ElementFactory.GetButton(By.XPath($"(//android.widget.ImageView[@resource-id='com.nextcloud.client:id/overflow_menu'])[{index}]"), "MoreMemu");
```